AUTHORS

```diff
@@ -1074,13 +1071,102 @@ Kunal Pathak <Kunal.Pathak@microsoft.com>
 Tracy Hinds <tracyhinds@gmail.com>
 lazlojuly <lazlo.gy@gmail.com>
 Arnaud Lefebvre <a.lefebvre@outlook.fr>
-not-an-aardvark <not-an-aardvark@users.noreply.github.com>
+not-an-aardvark <not-an-aardvark@>
 Junshu Okamoto <o_askgulf@icloud.com>
 Shahid Shaikh <rwtc66@gmail.com>
 Simen Bekkhus <sbekkhus91@gmail.com>
 Jason Hedrick <thecoolestguy@gmail.com>
 David Keeler <dkeeler@mozilla.com>
 Zwb <zwb.ict@gmail.com>
 Paul Grock <paulgrock@gmail.com>
+Hubert Mine <hubertmine@hotmail.fr>
+Dan Fabulich <dan.fabulich@redfin.com>
+Mike Ralphson <mike.ralphson@gmail.com>
+Alexis374 <879736822@qq.com>
+atstojanov <atstojanov@gmail.com>
+Thomas Hunter II <me@thomashunter.name>
+Christopher Dunavan <github@hypersprite.com>
+Peter Ogilvie <code@ogilvie.us.com>
+Dan Williams <daniel@chat.za.net>
+Teddy Katz <teddy.katz@gmail.com>
+Kalman Hazins <kalmanh@gmail.com>
+Ltrlg <ltrlg.wp@gmail.com>
+Dennis Schwartz <dennis@repositive.io>
+Yevgen Safronov <sejoker@gmail.com>
+Tobias Kahse <tobias.kahse@hpe.com>
+Sébastien Barbieri <seba@rtbf.be>
+Pavol Otcenas <pavol@otcenas.eu>
+Alessandro Metta <n0f3@outlook.com>
+Dany Shaanan <danyshaanan@gmail.com>
+Ishan Aditya <ishanaditya@gmail.com>
+Rachel <rachel.black@flashtalking.com>
+Jason Ginchereau <jasongin@microsoft.com>
+Paul Kiddie <paul@paulkiddie.com>
+Scott Stern <scott.stern06@gmail.com>
+Danny Guo <dannyguo91@gmail.com>
+Irina Shestak <shestak.irina@gmail.com>
+Matt Lang <matt@mediasuite.co.nz>
+Thomas van Lankveld <thomas@cell-0.com>
+Tarjei Husøy <git@thusoy.com>
+Wietse Venema <wvenema@xebia.com>
+Jonathan Prince <jonathan.prince@gmail.com>
+Fikret Burak Gazioglu <burakgazi@static.ip-217-149-135-181.signet.nl>
+delvedor <tommydelved@gmail.com>
+Jermaine Oppong <jeropp00@gmail.com>
+Richard Walker <digitalsadhu@gmail.com>
+matzavinos <matzavinos@workable.com>
+Alfred Cepeda <AlfredJCepeda@gmail.com>
+Niklas Ingholt <niklas@ingholt.com>
+rainabba <rainabba@gmail.com>
```

Comment:
This an appropriate way to communicate a change request to this PR? Better way to do it?